### PR TITLE
Create new test on home if enableProfile is false

### PR DESCRIPTION
**Pull request by ethantmanns**
If `enableProfile` is false, navigating to the root of the app will create a new test run under the `UNSET_PROFILE` profile ID and navigate to that test run.

One thing to consider is clicking on the logo will create a new test. Should we consider where the logo in the navbar links to?

### CODE REVIEW

File: OpenRDTReaderWeb/src/utils/AppConfig.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ const MasterConfig: AppConfig = {
   photoUploadEnabled: true,
 
   staticPhotoMLRecognitionEnabled: true,
+
+  enableProfiles: true,
```
Comment (ethantmanns) [inline OpenRDTReaderWeb/src/utils/AppConfig.ts:21]: Is this the default we want? I assume so. I just had the override set locally to false to test

File: OpenRDTReaderWeb/src/components/App/DashboardApp.tsx [hunk 2]
```diff
@@ -4,24 +4,65 @@
...
+        startingPath = newTestPath;
+      }
+    }
+
+    setStartingPath(startingPath);
+  };
+
+  useEffect(() => {
+    getStartingPage();
+  });
```
Comment (ethantmanns) [inline OpenRDTReaderWeb/src/components/App/DashboardApp.tsx:64]: I'll add `[auth, config]` here

**Comment by yrichard** [comment]
Ah, yes, I think the logo should create a new test run as well.
Could we go to a page which does only this: 
- Create a new testrunid
- redirect to the start page for testruns.

File: OpenRDTReaderWeb/src/utils/AppConfig.ts [hunk 3]
```diff
@@ -17,6 +17,8 @@ const MasterConfig: AppConfig = {
   photoUploadEnabled: true,
 
   staticPhotoMLRecognitionEnabled: true,
+
+  landingPage: "HOME",
```
Comment (yrichard) [inline OpenRDTReaderWeb/src/utils/AppConfig.ts:21]: let's put NEWTESTRUN here.

**Pull request merged**